.github/workflows/semantic-pull-request.yml

```
@@ -0,0 +1,19 @@
+name: "Semantic PR"
```

Lets match the naming that's been in use.
```suggestion
name: "Semantic Pull Request"
```